docs/packaging.rst

```diff
@@ -98,4 +98,4 @@ Examples
 
 * `arch linux <https://www.archlinux.org/packages/community/any/python-hypothesis/>`_
 * `fedora <https://admin.fedoraproject.org/pkgdb/package/rpms/python-hypothesis/>`_
-* `gentoo <https://packages.gentoo.org/packages/dev-python/hypothesis>`_ (slightly behind at the time of this writing)
+* `gentoo <https://packages.gentoo.org/packages/dev-python/hypothesis>`_
```

Comment:
Looks like it’s become very behind now, or have I missed something?

Reply:
The stable (green) version is Hypothesis 3.6 which is pretty old, but they did update the testing (yellow) version to 3.28.2 yesterday.  Either way, it's not slightly behind and that kind of note goes out of date really quickly anyway.